storage/errors.go

```diff
@@ -339,11 +339,62 @@ var (
 	// hash cannot be stored because it is a duplicate.
 	ErrDuplicateTransactionHash = errors.New("duplicate transaction hash")
 
+	ErrBlockGetFailed                  = errors.New("unable to get block")
+	ErrTransactionGetFailed            = errors.New("could not get transaction")
```

Comment:
nit: thoughts on rephrase error message so that we are consistent?
```suggestion
	ErrTransactionGetFailed            = errors.New("unable to not get transaction")
```

Reply:
👍